src/shape/bounding-box.js

```diff
@@ -293,6 +334,12 @@ Object.assign(pc, function () {
             );
         },
 
+        /**
+         * @function
+         * @name pc.BoundingBox#compute
+         * @description Compute the size of the AABB to encapsulate all specified vertices.
+         * @param {pc.Vec3[]} vertices - The vertices used to compute the new size for the AABB.
```

Comment:
I believe `vertices` is just an Array or Float32Array.

Reply:
My bad. Fixed.